webapps-schema/src/main/resources/schema/elasticsearch/create/template/tasklist-task.json

```diff
@@ -0,0 +1,123 @@
+{
+  "mappings": {
+    "dynamic": "strict",
+    "properties": {
+      "join": {
+        "type": "join",
+        "eager_global_ordinals": true,
+        "relations": {
+          "task": [
+            "taskVariable"
+          ],
+          "process": [
+            "processVariable",
+            "task"
+          ]
+        }
+      },
+      "flowNodeBpmnId": {
+        "type": "keyword"
+      },
+      "flowNodeInstanceId": {
+        "type": "keyword"
+      },
+      "partitionId": {
+        "type": "integer"
+      },
+      "completionTime": {
+        "format": "date_time || epoch_millis",
+        "type": "date"
+      },
+      "processInstanceId": {
+        "type": "keyword"
+      },
+      "id": {
+        "type": "keyword"
+      },
```

Comment:
❓ In Tasklist-list-view there was a taskId. I guess we don't need this?

Reply:
The `taskId` field has the same value as the `flowNodeBpmnId` which is the Activity ID, so we'll keep only one.